apps/storybook/src/unstable/datacom-scroll-button.stories.tsx

```diff
@@ -34,14 +34,17 @@ export const ScrollButtonWithExample = () => {
     width: 842px;
     height: 320px;
     display: flex;
+    flex-direction: column;
     background-color: #f0f0f0;
     align-items: center;
     margin: 2rem;
     justify-content: center;
     scroll-margin-top: 30px;
     datacom-scroll-button {
-      margin-top: 200px;
-      margin-right: 400px;
+      margin-top: 24px;
+      margin-bottom: 24px;
+      justify-content: center;
+      text-align: center;
```

Comment:
These two lines of CSS don't do anything. Please remove. (Feel free to verify on your end.)